Components/Button/Button.html

```diff
@@ -0,0 +1,131 @@
+                Buttons are also called as call to action. We have range of buttons and their states. You may use a tag
+                or button element, you need to add respective classes, and you are good to go
+            </div>
+            <hr>
+
+            <!-- Contained Buttons -->
+            <h2 class="Content__TopicHeading">Contained Buttons</h2>
+            <div class="Content__TopicSubtitle">
+                Contained buttons are high-emphasis, distinguished by their use of elevation and fill. They contain
+                actions that are primary to your app.
+            </div>
+
+            <div class="ExampleBox">
+                <a href="" class="btn btn--Primary">Primary </a>
+                <a href="" class="btn btn--Secondary">Secondary </a>
+                <a href="" class="btn btn--Error">Error </a>
+                <a href="" class="btn btn--Success">Success </a>
+            </div>
+            <div class="Code-Snippet-Display">
+                <iframe
+                src="https://carbon.now.sh/embed/syBzxbr7wFRNlS2Cgo46"
+                style="width: 730px; height: 149px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+              </iframe>
+            </div>
+            <hr>
+
+             <!-- Contained Buttons -->
+             <h2 class="Content__TopicHeading">Outlined Buttons</h2>
+             <div class="Content__TopicSubtitle">
+                In need of a button, but not the hefty background colors they bring? Replace '.btn-variant' with '.outline-variant'.
+             </div>
+
+             <div class="ExampleBox">
+                <a href="" class="btn btn--Primary-Outlined">Primary </a>
+                <a href="" class="btn btn--Secondary-Outlined">Secondary </a>
+                <a href="" class="btn btn--Error-Outlined">Error </a>
+                <a href="" class="btn btn--Success-Outlined">Success </a>
+            </div>
+            <div class="Code-Snippet-Display">
+                <iframe
+                src="https://carbon.now.sh/embed/syBzxbr7wFRNlS2Cgo46"
+                style="width: 730px; height: 149px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+              </iframe>
+            </div>
+            <hr>
+
+            <!-- Link Buttons -->
+            <h2 class="Content__TopicHeading">Link Button</h2>
+            <div class="Content__TopicSubtitle">
+                Want to attach a link action, at a button tap? Use Link Button.
+            </div>
+
+            <div class="ExampleBox">
+                <a href="Link" class="btn btn--Primary-Link">Link </a>
+            </div>
+            <div class="Code-Snippet-Display">
+                <iframe
+                src="https://carbon.now.sh/embed/syBzxbr7wFRNlS2Cgo46"
+                style="width: 730px; height: 111px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
Avoid inline style tags

Reply:
They come while copying the iframe itself from the carbon sh website.

Reply:
Ok, understood